docs/orchestrating-elastic-stack-applications/elasticsearch/http-settings-tls-sans.asciidoc

```diff
@@ -21,7 +21,7 @@ spec:
 
 Check the https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types[Kubernetes Publishing Services (ServiceTypes)] that are currently available.
 
-NOTE: Please note that when you change the `clusterIP` setting of the service, ECK will delete and re-create the service as `clusterIP` is an immutable field. Depending on your client implementation this might have an a short impact on the ability to resolve the service's DNS name to endpoints.
+NOTE: When you change the `clusterIP` setting of the service, ECK will delete and re-create the service, as `clusterIP` is an immutable field. Depending on your client implementation, this might have an impact on the ability to resolve the service's DNS name to endpoints.
```

Comment:
I think we do want the "short" qualifier or something else there to indicate that it is a temporary condition, right? Otherwise I could read this now and think that it might permanently break it

Also I realize that this part
>NOTE: When you change the `clusterIP` setting of the service,

Is a little confusing. What we are changing is the `spec.http.service.spec.type` field from its previous value of `ClusterIP`

Reply:
How about _temporary impact_ tout court?

Reply:
> What we are changing is the spec.http.service.spec.type field from its previous value of ClusterIP

I think this was about users providing a fixed ClusterIP that is not the one that currently exists. In which case we recreate the service entirely to get rid of the previous ClusterIP.

Reply:
Possible rewording:

> When you change the `ClusterIP` value in the `spec.http.service.spec.type` field,  ECK will delete and re-create the service, as `clusterIP` is an immutable field.